src/wp-includes/class-walker-nav-menu.php

```diff
@@ -212,8 +213,25 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 
 		$output .= $indent . '<li' . $li_attributes . '>';
 
+		/** This filter is documented in wp-includes/post-template.php */
+		$title = apply_filters( 'the_title', $menu_item->title, $menu_item->ID );
+
+		// Save filtered value before filtering again.
+		$title_filtered = $title;
```

Comment:
I did not want to change the `$title` variable name in the `nav_menu_item_title` filter, so it just assigns the new variable for use later. The `$title_filtered` name probably can be improved (it's after one filter but not all).

Reply:
Maybe `$the_title_filtered`, using the filter name in the variable?